client/reader/sidebar/promo.jsx

```diff
@@ -23,7 +23,7 @@ export const ReaderSidebarPromo = ( { currentUserLocale, shouldRenderAppPromo }
 
 			{ shouldRenderAppPromo && (
 				<div className="sidebar__app-promo">
-					<AppPromo location="reader" locale={ currentUserLocale } />
+					<AsyncLoad require="blocks/app-promo" location="reader" locale={ currentUserLocale } />
```

Comment:
Should we make that the default? It seems like it comes up a lot when we’re loading components async.

Reply:
There's a good reason to have this placeholder when something's dynamically loading - often it is useful to know that something is loading. But for components that may not display anything, it might be a false alarm - one that could be confusing to the user.